### ISSUE TITLE
Migrated old plugin description appears twice

### ISSUE THREAD

**Issue by tompidev** [bludit/bludit#1382]
Hello,

For plugins migrated from the old version, the plugin description section appears twice at the top of plugin configuration page.
This section is already specified directly in the kernel file.

(/bl-kernel/admin/views/plugins-settings.php)

```php
if ($plugin->description()) {
echo '<div class="alert alert-primary" role="alert">'.$plugin->description().'</div>';
}
```
**To fix this error:**
The following section of the `plugin.php` file should be deleted:

```php
$html  = '<div class="alert alert-primary" role="alert">';
$html .= $this->description();
$html .= '</div>';
```

**But remember!** 

Within a function, the first `$html` string does not need a dot before the equal sign! (see code above)

---

***Bludit version:** v4.0.0 Beta 2*

***PHP version:** PHP 7.4.x*

**Comment by dignajar** [comment]
Fixed, but I still have to fix the CSS for the half of the plugins..

**Comment by dignajar** [comment]
Fixed, please check the new release v4.0.0 rc3.
Feel free to open the ticket if the issue is still there, if a new issue please open a new ticket.
